src/python/pants/backend/adhoc/adhoc_tool.py

```diff
@@ -71,104 +59,37 @@ async def run_in_sandbox_request(
+            runnable_dependencies=target.get(AdhocToolRunnableDependenciesField).value or (),
+            target=request.protocol_target,
         ),
     )
 
-    addresses = Addresses((runnable_address,))
-    addresses.expect_single()
-
-    runnable_targets = await Get(Targets, Addresses, addresses)
-    field_sets = await Get(
-        FieldSetsPerTarget, FieldSetsPerTargetRequest(RunFieldSet, runnable_targets)
-    )
-    run_field_set: RunFieldSet = field_sets.field_sets[0]
-
     working_directory = target[AdhocToolWorkdirField].value or ""
     root_output_directory = target[AdhocToolOutputRootDirField].value or ""
 
-    # Must be run in target environment so that the binaries/envvars match the execution
-    # environment when we actually run the process.
-    run_request = await Get(
-        RunInSandboxRequest, {environment_name: EnvironmentName, run_field_set: RunFieldSet}
-    )
-
-    execution_environment = await Get(
-        ResolvedExecutionDependencies,
-        ResolveExecutionDependenciesRequest(
-            target.address,
-            target.get(AdhocToolExecutionDependenciesField).value,
-            target.get(AdhocToolRunnableDependenciesField).value,
-        ),
-    )
-    dependencies_digest = execution_environment.digest
-    runnable_dependencies = execution_environment.runnable_dependencies
-
-    extra_env: dict[str, str] = dict(run_request.extra_env or {})
-    extra_path = extra_env.pop("PATH", None)
-
-    extra_sandbox_contents = []
-
-    extra_sandbox_contents.append(
-        ExtraSandboxContents(
-            EMPTY_DIGEST,
-            extra_path,
-            run_request.immutable_input_digests or FrozenDict(),
-            run_request.append_only_caches or FrozenDict(),
-            run_request.extra_env or FrozenDict(),
-        )
-    )
-
-    if runnable_dependencies:
-        extra_sandbox_contents.append(
-            ExtraSandboxContents(
-                EMPTY_DIGEST,
-                f"{{chroot}}/{runnable_dependencies.path_component}",
-                runnable_dependencies.immutable_input_digests,
-                runnable_dependencies.append_only_caches,
-                runnable_dependencies.extra_env,
-            )
-        )
-
-    merged_extras = await Get(
-        ExtraSandboxContents, MergeExtraSandboxContents(tuple(extra_sandbox_contents))
-    )
-    extra_env = dict(merged_extras.extra_env)
-    if merged_extras.path:
-        extra_env["PATH"] = merged_extras.path
-
-    input_digest = await Get(Digest, MergeDigests((dependencies_digest, run_request.digest)))
-
     output_files = target.get(AdhocToolOutputFilesField).value or ()
     output_directories = target.get(AdhocToolOutputDirectoriesField).value or ()
 
-    extra_args = target.get(AdhocToolArgumentsField).value or ()
-
-    append_only_caches = {
-        **merged_extras.append_only_caches,
-        **(target.get(AdhocToolNamedCachesField).value or {}),
-    }
```

Comment:
I think this section got dropped

Reply:
ack! you are right, will fix. Thank you for spotting it.